rent_golem/backend/rent_golem/rent_golem/settings.py

```diff
@@ -44,7 +43,8 @@
     "django_celery_results",
     "rest_framework",
 
-    "clusters"
+    "clusters",
+    "accounts"
```

Comment:
I prefer to add `,` at the end of elements organized in columns because diffs look better. You can see the example here. Previously `cluster` had no comma at the end and now is highlighted as change just because there is comma added.

Reply:
Done